recipes/r-captioner/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = '2.2.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-captioner
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: captioner_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/captioner_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/captioner/captioner_{{ version }}.tar.gz
+  sha256: 72f92d757a47bae883cc95eb60a115c141aaf5eddcc15bc8068ea0faea15e775
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('captioner')"  # [not win]
+    - "\"%R%\" -e \"library('captioner')\""  # [win]
+
+about:
+  home: https://github.com/adletaw/captioner
+  license: Custom
+  license_url: https://github.com/adletaw/captioner/LICENSE
```

Comment:
Can you download the file and add it to the recipe? See https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
Thanks for the feedback @ocefpaf.  I've changed the license to `Other` and linked in the license file as described at that link.  Unfortunately the "license" isn't much of a license.  The entire text is:

```
YEAR: 2015
COPYRIGHT HOLDER: Alathea D Letaw
```

but there's not much I can do about that.

Reply:
:confounded: That is always a problem. There is no clarity if we can package and ship that or not :unamused:

Reply:
I was worried you might say that.  It _is_ redistributed via CRAN but I can't say if thats good enough.  I just opened an issue with the author to request they select a recognized license, and offered to help with that.  No idea what will come of that though.

Reply:
> It _is_ redistributed via CRAN

Is the author the one that put it there? Sorry, but that does not say much.

>  I just opened an issue with the author to request they select a recognized license, and offered to help with that.

Awesome! Thanks!!

> No idea what will come of that though.

Let's wait  for @bgruening here. He probably knows more about the R-ecosystem than me.